src/cockpit/router.py

```diff
@@ -26,12 +27,47 @@
 logger = logging.getLogger(__name__)
 
 
+class ExecutionQueue(collections.deque):
+    def run(self):
+        for method, args in self:
+            method(*args)
+
+    def wrap(self, method):
+        """Temporarily delay calls to a given class method.
+
+        Functions by replacing the named function at the instance __dict__
+        level, effectively providing an override for exactly one instance.
```

Comment:
That's the object (i.e. class instance) that contains the wrapped function, not the ExecutionQueue instance, right?

```suggestion
        level, effectively providing an override for exactly one instance
        of `method`'s object.
```

Reply:
Correct.  Accepting your change.